src/stores/widgetManager.ts

```diff
@@ -409,14 +410,16 @@ export const useWidgetManagerStore = defineStore('widget-manager', () => {
     const container: MiniWidgetContainer | undefined = miniWidgetContainersInCurrentView.value.find((cont) => {
       return cont.widgets.includes(miniWidget)
     })
-
     if (container === undefined) {
       Swal.fire({ icon: 'error', text: 'Mini-widget container not found.' })
       return
     }
 
     const index = container.widgets.indexOf(miniWidget)
     container.widgets.splice(index, 1)
+
+    // Remove miniWidget variable from the list of currently logged variables
+    CurrentlyLoggedVariables.removeVariable(miniWidget.options.displayName)
```

Comment:
I'm not sure. I feel like this does not belong here. This is the widget store, but here we are coupling it with the logging system.

Can't we do this on the unmount of the widget itself?

Reply:
For the time, so we do not hurt the roadmap, we decide to let this coupling how it is and open an issue to track a future improvement on it.